test/api/liskApi.js

```diff
@@ -550,4 +535,38 @@ describe('Lisk.api()', function () {
 			});
 		});
 	});
-});
+
+	describe('#getAccount', function() {
+		it('should get account information', function (done) {
```

Comment:
`should get account information`

Maybe: `should get an account using a given address`. Please review sibling descriptions for: `#getBlock`, `#getTransaction`.